SConstruct

```diff
@@ -345,81 +343,43 @@ config_options = [
         'python_array_home',
         """If NumPy was installed using the '--home' option, set this to the home
-           directory for NumPy for Python 2.""",
+           directory for NumPy.""",
         '', PathVariable.PathAccept),
     PathVariable(
         'python_prefix',
-        """Use this option if you want to install the Cantera Python 2 package to
+        """Use this option if you want to install the Cantera Python package to
            an alternate location. On Unix-like systems, the default is the same
            as the 'prefix' option. If the 'python_prefix' option is set to
            the empty string or the 'prefix' option is not set, then the package
            will be installed to the system default 'site-packages' directory.
            To install to the current user's 'site-packages' directory, use
            'python_prefix=USER'.""",
         defaults.python_prefix, PathVariable.PathAccept),
-    EnumVariable(
-        'python2_package',
-        """Controls whether or not the Python 2 module will be built. By
-           default, the module will be built if the Python 2 interpreter
-           and the required dependencies (NumPy for Python 2 and Cython
-           for the version of Python for which SCons is installed) can be
-           found.""",
-        'default', ('full', 'minimal', 'y', 'n', 'none', 'default')),
-    PathVariable(
-        'python2_cmd',
-        """The path to the Python 2 interpreter. The default is
-           'python2'; if this executable cannot be found, this
-           value must be specified to build the Python 2 module.""",
-        'python2', PathVariable.PathAccept),
-    PathVariable(
-        'python2_array_home',
-        """If NumPy was installed using the '--home' option, set this to the home
-           directory for NumPy for Python 2.""",
-        '', PathVariable.PathAccept),
-    PathVariable(
-        'python2_prefix',
-        """Use this option if you want to install the Cantera Python 2 package to
-           an alternate location. On Unix-like systems, the default is the same
-           as the 'prefix' option. If the 'python_prefix' option is set to
-           the empty string or the 'prefix' option is not set, then the package
-           will be installed to the system default 'site-packages' directory.
-           To install to the current user's 'site-packages' directory, use
-           'python2_prefix=USER'.""",
-        defaults.python_prefix, PathVariable.PathAccept),
     EnumVariable(
         'python3_package',
-        """Controls whether or not the Python 3 module will be built. By
-           default, the module will be built if the Python 3 interpreter
-           and the required dependencies (NumPy for Python 3 and Cython
-           for the version of Python for which SCons is installed) can be
-           found.""",
+        """Deprecated synonym for the 'python_package' option. Will be overridden
+        if 'python_package' is set.""",
```

Comment:
We should be consistent about using \`\`option_name`` or 'option_name' in these strings. I would prefer the former, to make auto-generation of the corresponding docs page easier. That said, it seems like most of them use the latter form, and it makes the terminal-formatted `scons help` look nicer.

Reply:
Maybe this is just something that should just be handled when https://github.com/Cantera/cantera-website/issues/26 is resolved

Reply:
I'm 👍 on deferring this to that other issue, given that ``'option_name'`` is currently used in all but two options.